docsrc/hello_world.rst

```diff
@@ -61,88 +54,90 @@ By default, the Stan program is compiled on instantiation.
+either as a filepath or a Python dictionary; in this example, we use the
 example datafile `bernoulli.data.json`:
 
+By default, the ``sample`` method runs 4 sampler chains.
+The ``output_dir`` argument is an optional argument which specifies
+the path to the output directory used by CmdStan.
+If this argument is omitted, the output files are written
+to a temporary directory which is deleted when the current Python session is terminated. [#]_
 
-By default, the ``sample`` command runs 4 sampler chains.
-This is a set of per-chain 
-`Stan CSV files <https://mc-stan.org/docs/cmdstan-guide/stan-csv.html#mcmc-sampler-csv-output>`__
-The filenames follow the template '<model_name>-<YYYYMMDDHHMM>-<chain_id>'
-plus the file suffix '.csv'.
-There is also a correspondingly named file with suffix '.txt'
-which contains all messages written to the console.
-If the ``output_dir`` argument is omitted, the output files are written
-to a temporary directory which is deleted when the current Python session is terminated.
 
 .. code-block:: python
 
     # specify data file
     bernoulli_data = os.path.join(cmdstan_path(), 'examples', 'bernoulli', 'bernoulli.data.json')
 
     # fit the model 
-    bern_fit = bernoulli_model.sample(data=bernoulli_data, output_dir='.') 
+    bernoulli_fit = bernoulli_model.sample(data=bernoulli_data, output_dir='.') 
 
     # printing the object reports sampler commands, output files
-    print(bern_fit)
-
+    print(bernoulli_fit)
 
-Access the sample
-^^^^^^^^^^^^^^^^^
 
-The :ref:`class_cmdstanmcmc` object
-provides properties and methods to access, summarize, and manage the sample and its metadata.
+Accessing the sample
+^^^^^^^^^^^^^^^^^^^^
 
-The sampler and model outputs from each chain are written out to Stan CSV files.
-The CmdStanMCMC object assembles these outputs into a
-numpy.ndarray which contains all across all chains arranged as (draws, chains, columns). 
-The ``draws`` method returns the draws array.
-By default, it returns the underlying 3D array.
-The optional boolean argument ``concat_chains``, when ``True``,
-will flatten the chains resulting in a 2D array.
+The CmdStan `sample` method outputs are a set of per-chain
+`Stan CSV files <https://mc-stan.org/docs/cmdstan-guide/stan-csv.html#mcmc-sampler-csv-output>`__.
+The filenames follow the template '<model_name>-<YYYYMMDDHHMM>-<chain_id>'
+plus the file suffix '.csv'.
+The :class:`cmdstanpy.CmdStanMCMC` class provides methods to assemble the contents
+of these files in memory as well as methods to manage the disk files.
+
+Underlyingly, the draws from all chains are stored as an
+a numpy.ndarray with dimensions: draws, chains, columns.
+CmdStanPy provides accessor methods which return the sample
+either in terms of the CSV file columns or in terms of the
+sampler and Stan program variables.
+The ``draws`` and ``draws_pd`` methods return the sample contents
+in columnar format.
+
+The ``stan_variable`` method to returns a numpy.ndarray object
```

Comment:
instead of 

```
``stan_variable``
```

we should be using 

```
:py:meth:`~cmdstanpy.CmdStanMCMC.stan_variable`
```

this renders the exact same but creates a link to the autosummary of the method function. We should do this for essentially __every__ function name in the doc I believe. I'm happy to do so if you don't want to, it's rather tedious

Reply:
Ah, I see further down you're doing this on the workflow page. The only difference is the `~` character, which removes the package/class from the text.

e.g.

cmdstanpy.write_stan_json renders as **``cmdstanpy.write_stan_json()``**

~cmdstanpy.write_stan_json renders as **``write_stan_json()``**